blur.rb

```diff
@@ -0,0 +1,124 @@
+class Image
```

Comment:
Wow, can't believe that I missed option 1! It didn't occur to me at first b/c of my `changed` property on Pixel (I'd have to reset it every time) but this is totally doable.

I feel like option 2 would be relatively easy to do with the `.permutation` method.

I'll work on building out versions of these other two options.